lib/reporter.js

```diff
@@ -16,6 +16,7 @@ var generateReport = function (options) {
     var packageJson = {};
     try {
         packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+        packageJson.name = packageJson.name.replace('@','__').replace('/','_'); // sanitize scoped packages
```

Comment:
We are sanitizing the string at more than one places; will generalize it & sanitize the `suite.name` since it's been passed to the HTML templates.